src/plugin.py

```diff
@@ -214,10 +220,10 @@ async def get_game_time(self, game_id, context):
         else:
             multimc_time = GameTime(game_id, 0, None)
         time = tracked_time.time_played + multimc_time.time_played
-        lastPlayed = utils.compare(tracked_time.last_played_time, multimc_time.last_played_time)
+        lastPlayed = misc.compare(tracked_time.last_played_time, multimc_time.last_played_time)
         log.debug(f"Got game time: {time}")
         if time == 0 or lastPlayed is None:
-            return None
+            return GameTime(game_id, None, None)

```

Comment:
why just not `return GameTime(game_id, time, lastPlayed)` without condition `time == 0 or lastPlayed is None:`?

is it somehow invalid scenario if only one of those values is None?

Reply:
Yeah, I think you're right I don't think it is necessary. I'll make this change and [this one](https://github.com/FriendsOfGalaxy/galaxy-integration-minecraft/pull/8#discussion_r490075587) quickly and add it to [the other pull request](https://github.com/TouwaStar/Galaxy_Plugin_Minecraft/pull/11). It won't be included in this release but it should be in the next release once more changes have been made as the current changes are minimal and don't really do anything.